components/text_utilities.css

```diff
@@ -0,0 +1,66 @@
+.text-left-align{
+    text-align: left;
+}
+
+.text-center-align{
+    text-align: center;
+}
+
+.text-right-align{
+    text-align: right;
+}
+
+.text-decorations{
+    margin-top: 0rem 1rem;
```

Comment:
Please check this property value.I think it should be 
margin-top : 0rem   OR margin-top : 1rem

It will create a conflict i guess.Please check

Reply:
Yeah, Totally missed this part.